content/ko/blog/_posts/2025-09-02-prefer-align-cpus-by-uncore-cache-beta/index.md

```diff
@@ -0,0 +1,89 @@
+---
+layout: blog
+title: '쿠버네티스 v1.34: CPU Manager Static Policy의 Uncore Cache 정렬 옵션 도입'
+date: 2025-09-02T10:30:00-08:00
+slug: kubernetes-v1-34-prefer-align-by-uncore-cache-cpumanager-static-policy-optimization
+author: Charles Wong (AMD)
+translator:
+  변재한 (수퍼게이트)
+---
+
+Kubernetes v1.32에서 prefer-align-cpus-by-uncorecache라는 새로운 CPU Manager Static Policy 옵션이 알파 기능으로 처음 도입되었고, Kubernetes v1.34에서는 베타 단계로 승격되었습니다. 이 옵션은 분할된 언코어 캐시 아키텍처를 사용하는 프로세서에서 실행되는 특정 워크로드의 성능을 최적화하도록 설계되었습니다. 이 글에서는 해당 개념이 무엇을 의미하는지, 그리고 왜 유용한지를 설명합니다.
+
+## 기능 이해하기
+
+### Uncore 캐시란 무엇인가?
+얼마 전까지 대부분의 범용 CPU는 여러 코어가 단일(last-level) L3 캐시를 공유하는 구조였습니다. 이 단일 L3 캐시는 특정 코어에 연결되지 않기 때문에 _언코어(Uncore) 캐시(=L3 캐시)_ 라고 부르기도 합니다. 반대로 L1, L2 캐시는 보통 특정 코어에 종속되어 있습니다.
+
+최근의 AMD64 및 ARM 기반 프로세서들은 코어와 캐시 간 접근 지연(latency)을 줄이기 위해 분할된 Uncore 캐시 아키텍처를 도입했습니다. 즉, 최종 레벨 캐시(L3)를 물리적으로 나눠 CPU 패키지 내 특정 CPU 그룹과 정렬(alignment)되도록 구성합니다. 이렇게 패키지 내부 거리를 줄이면 캐시 접근 지연을 줄일 수 있습니다.
+
+![Diagram showing monolithic cache on the left and split cache on the right](./mono_vs_split_uncore.png)
+
+쿠버네티스는 이러한 CPU 패키지(들) 내 캐시 토폴로지를 고려해 워크로드를 배치할 수 있습니다.
+
+### 캐시 인식(Cache-aware) 워크로드 배치
+
+아래의 매트릭스는 분할된 언코어 캐시를 사용하는 프로세서에서, 캐시 일관성(coherence) 프로토콜을 통해 CPU 간에 패킷을 전달할 때의 [CPU↔CPU 지연 시간(ns)](https://github.com/nviennot/core-to-core-latency)을 보여줍니다(낮을수록 좋음). 이 예시에서 하나의 프로세서 패키지는 2개의 언코어 캐시를 가지며, 각 언코어 캐시는 8개의 CPU 코어를 담당합니다.
+
+![CPU-to-CPU 지연 시간 수치를 보여주는 표](./c2c_latency.png)
+
+- 파란 칸: 같은 언코어 캐시를 공유하는 CPU 간 지연(낮음)
+- 회색 칸: 서로 다른 언코어 캐시에 속한 CPU 간 지연(높음)
+
+`prefer-align-cpus-by-uncorecache`를 활성화하면 [static CPU Manager](/docs/concepts/policy/node-resource-managers/#static-policy)는 컨테이너에 할당되는 모든 CPU가 가능한 한 동일한 언코어 캐시를 공유하도록 CPU 리소스를 할당하려고 시도합니다. 이 정책은 best-effort 방식으로 동작하며, 컨테이너의 요구사항과 노드의 가용 리소스를 함께 고려해 컨테이너의 CPU가 여러 언코어 캐시에 흩어지는 것을 최소화합니다.
+
+가능한 가장 적은 수의 언코어 캐시를 사용하는 CPU 집합 위에서 워크로드를 실행하면, (위 매트릭스에서처럼) 캐시 지연이 감소하고 타 워크로드와의 캐시 경합도 줄어들어 _전체 처리량(throughput)_ 이 향상될 수 있습니다. 단, 이러한 이점은 노드의 프로세서가 분할 언코어 캐시 토폴로지일 때에만 나타납니다.
+
+아래 다이어그램은 기능 활성화 시의 언코어 캐시 정렬을 보여줍니다.
+
+![Diagram showing an example workload CPU assignment, default static policy, and with prefer-align-cpus-by-uncorecache](./cache-align-diagram.png)
```

Comment:
Plz translate "Diagram showing an example workload CPU assignment, default static policy, and with prefer-align-cpus-by-uncorecache"